hawk/src/main/java/com/orhanobut/hawk/HawkBuilder.java

```diff
@@ -27,13 +33,21 @@
   private static final String KEY_NO_CRYPTO = "dfsklj2342nasdfoasdfcrpknasdf";
 
   private Context context;
+
```

Comment:
Can we remove the new added spaces please for the convention consistency? I favor spaces when there is javadoc, otherwise I think there is no need.